doc/deprecation_process.md

```diff
@@ -2,57 +2,160 @@ This page can be linked using: [aka.ms/azsdk/python/deprecation-process](https:/
 
-- `CHANGELOG.MD` add a new version with the current date, and the same disclaimer. For instance
+> # Microsoft Azure SDK for Python
+>
+> This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
+>
+> For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
 
-   > ## 1.2.3.post1 (2023-03-31)
-   >
-   > This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
-   >
-   > For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
+**NOTE**: While a migration guide should always be written, you may decide to postpone this work based on downloads numbers (found on [pypistats](https://pypistats.org/), [pype.tech](https://www.pepy.tech/), etc.) and internal knowledge of the usage of the package.
+
+## CHANGELOG.md and _version.py
+
+In `CHANGELOG.md`, add a new version with a disclaimer in the following format. The version should be the next beta or patch version. For example:
+
+If the last released version was 1.2.3b1, the version should be 1.2.3b2.
+
+> ## 1.2.3b2 (2023-03-31)
+>
+> ### Other Changes
+>
+> - This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
+> - For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
    
-   or
+If the last released version was 1.2.3, the version should be 1.2.4.
+
+> ## 1.2.4 (2023-03-31)
+>
+> ### Other Changes
+>
+> - This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
+> - For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
+
+Update the version in the `azure/mypackage/_version.py` file. This file may be called `version.py` if your package is very old.
+
+More information on specifics of versioning can be found at [the bottom of the guide under "More details - Versioning"](#versioning).
+
+## sdk_packaging.toml
+
+- Add `auto_update = false` if not already present to avoid the bot overriding your changes.
+
+## pyproject.toml
+
+- Ensure `ci_enabled = false` is NOT present in pyproject.toml. If it is, remove the line, as this will prevent you from releasing the package.
+
+# Step 2: Resolve all open issues/PRs corresponding to the library.
+
+If there is a Track 2, provide a link to the new package or an existing migration guide before closing.
+
+# Step 3: Create a PR
+
+Create a PR targeting the `main` branch.
+
+Example PR to deprecate azure-cognitiveservices-language-luis [here.](https://github.com/Azure/azure-sdk-for-python/pull/36893/files)
+
+## Fix any CI issues
+
+Wait for the CI to run. Fix any issues related to the PR.
```

Comment:
Do we need to fix everything? I could imagine we're deprecating the package, and therefore we don't care about fixing tests or mypy?

Reply:
no, updating to say they should only fix what's needed for deprecation.